astropy/table/table.py

```diff
@@ -1792,6 +1792,12 @@ def _is_list_or_tuple_of_str(names):
     def keys(self):
         return list(self.columns.keys())
 
+    def values(self):
+        list_to_return = []
+        for i in self.as_array():
```

Comment:
I feel like this can use `tolist()` method in Numpy array and also be shortened into a one-liner using list comprehension. But I am deferring to `table` maintainers.

Reply:
@mhvk is the outptut wrong should it give output like {column_name:[elements of the column]},
right now the ouptut is like [[elements of each column except the column name]]

Reply:
The output should just be identical to `itercols`.

Reply:
@mhvk you mean that .values should return a generator, if so then wouldn't it introduce redundancy since itercols also returns a generator.